users/api/clients.go

```diff
@@ -139,6 +126,22 @@ func clientsHandler(svc users.Service, r *chi.Mux, logger *slog.Logger, pr *rege
 		), "disable_client").ServeHTTP)
 	})
 
+	r.Route("/password", func(r chi.Router) {
+		r.Post("/reset-request", otelhttp.NewHandler(kithttp.NewServer(
+			passwordResetRequestEndpoint(svc),
+			decodePasswordResetRequest,
+			api.EncodeResponse,
+			opts...,
+		), "password_reset_req").ServeHTTP)
+
+		r.Put("/reset", otelhttp.NewHandler(kithttp.NewServer(
+			passwordResetEndpoint(svc),
+			decodePasswordReset,
+			api.EncodeResponse,
+			opts...,
+		), "password_reset").ServeHTTP)
+	})
+
```

Comment:
Why we are going to change these paths ?

Reply:
From SDK it is under` /password` that is why I made the change to API docs and code base https://github.com/absmach/magistrala/blob/364749f28182d4e4a7a317e605f38c5c2b6aa222/pkg/sdk/go/users.go#L204

Reply:
We agreed and reverted to using non users prefix on password endpoint